host_core/native/hostcore_wasmcloud_native/src/wasmruntime.rs

```diff
@@ -60,35 +71,74 @@ impl Handle<HostInvocation> for ElixirHandler {
         //let caller_token = set_caller(...);
         //let caller_token = 12;
 
-        let op = invocation.operation.clone();
-        let mut msg_env = OwnedEnv::new();
-        msg_env.send_and_clear(&self.pid.clone(), |env| {
-            (
-                atoms::invoke_callback(),
-                crate::Claims::from(claims.clone()),
-                binding,
-                invocation.namespace,
-                invocation.operation,
-                invocation.payload.unwrap_or_default(),
-                callback_token.clone(),
-            )
-                .encode(env)
-        });
+        match invocation {
+            Invocation::Logging(LoggingInvocation::WriteLog { level, text }) => {
+                let level = match level {
+                    logging::Level::Debug => "debug",
+                    logging::Level::Info => "info",
+                    logging::Level::Warn => "warn",
+                    logging::Level::Error => "error",
+                };
+                trace!(level, subject = claims.subject, text);
```

Comment:
Once the instance of the logger implementation is correlated to a specific virtual host, you can add the value of the `host_id` field to the log output